sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -103,6 +104,9 @@ private[ui] abstract class ExecutionTable(
       <td>
         {descriptionCell(executionUIData)}
       </td>
+      <td>
+        {executionUIData.sqlText.getOrElse("-")}
```

Comment:
I've just checked and found out that the common practice in the Spark Web UI is to use a blank instead of a hyphen. I'll fix. Thanks!